{{ cookiecutter.repo_name }}/docs/pull_merge_request_templates/GDS.md

```diff
@@ -0,0 +1,32 @@
+# Summary
+
+Add your summary here - keep it brief, to the point, and in plain English.
```

Comment:
Can you give a good example? or would that clutter

Reply:
Might be clutter, but also might be useful! I'm thinking something like this:

> I've written a script to make predictions from any pre-trained scikit-learn classifier using data from a Google 
> BigQuery database.
>
> This enables us to classify unseen data as it comes in, returning a binary prediction for each session. We can use 
> these predictions to segment the data for further downstream intervention.
>
> The script is written in Python, and accepts **any** scikit-learn-based classifier. It includes additional scripts to 
> load data from the database, perform simple pre-processing for the classifier, and output the predictions as a CSV 
> file.
>
> Tests have been written for all the functions in the script; these are mainly in data loading and pre-processing. I 
> have assumed that all strings are in ASCII format. Performance tests on the chosen classifier are outside of scope, 
> and it is assumed to be a correctly tuned classifier.
>
> The majority of the scripts are straightforward, but I would like the reviewer to focus mainly on the pre-processing 
> steps, as the existing ones used in training the classifier have been refactored for performance.

I can either add it as HTML comments (so you won't see it unless you're looking at the raw Markdown), or directly in Markdown.

Alternatively, to reduce clutter, I could modify the Markdown to refer to the [GDS Way](https://gds-way.cloudapps.digital/standards/pull-requests.html) on pull requests if you think that's sufficient?

Reply:
Bit of a cluttered approach, so have instead referenced the [GDS Way](https://gds-way.cloudapps.digital/standards/pull-requests.html) on pull requests in commit [`90fff22`](https://github.com/ukgovdatascience/govcookiecutter/pull/6/commits/90fff22ff0d429da1586d98ba2862029042ec584).

Marking as **Resolved**.